src/components/FlowRunListItem.vue

```diff
@@ -65,8 +65,11 @@
   const tags = computed(() => props.flowRun.tags)
   const value = computed(() => props.flowRun.id)
 
-  const flowRunId = computed(() => props.flowRun.id)
-  const { taskRunsCount } = useTaskRunsCount(flowRunId)
+  const { count: taskRunsCount } = useTaskRunsCount(() => ({
```

Comment:
Same here wrt anonymous function not helping readability

Reply:
Talked about this in slack. In github it is kinda hard to parse. But in vscode it looks nice. For now we'll keep these but might revisit

<img width="456" alt="image" src="https://github.com/PrefectHQ/prefect-ui-library/assets/6200442/e2dbfab9-555d-4db1-9b31-fee455372fdd">